scripts/load_data.sh

```diff
@@ -5,9 +5,16 @@ apk -q --no-cache add curl jq
 
 pip -q install s3cmd
 
-for file in NA12878.bam NA12878_20k_b37.bam; do
+FILES="NA12878.bam NA12878.bai NA12878_20k_b37.bam NA12878_20k_b37.bai"
+for file in ${FILES}; do
```

Comment:
When `FILES` is an array:
```suggestion
for file in "${FILES[@]}"; do
```